Library/Formula/pixz.rb

```diff
@@ -17,17 +18,22 @@ class Pixz < Formula
   end
 
   def install
+    ENV["PKG_CONFIG_PATH"] = "#{HOMEBREW_PREFIX}/lib/pkgconfig:#{HOMEBREW_PREFIX}/opt/libarchive/lib/pkgconfig"
```

Comment:
I will take a look at that. Anything else? Would prefer to make any additional changes all at once.